pymeasure/instruments/instrument.py

```diff
@@ -145,7 +147,9 @@ class Instrument:
     __reserved_prefix = "___"
 
     # noinspection PyPep8Naming
-    def __init__(self, adapter, name, includeSCPI=True, **kwargs):
+    def __init__(self, adapter, name, includeSCPI=True, query_delay=0,
+                 preprocess_reply=None, **kwargs):
+        """Initialize the instrument."""
```

Comment:
you should not need to add such a docstring -- if a docstring is present on `class`, that one is taken for init

Reply:
I did not know this. Thanks for pointing out.